TILApp/Views/CommunityView/WebViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  WebViewController.swift
+//  TILApp
+//
+//  Created by Lee on 10/27/23.
+//
+
+import FlexLayout
+import PinLayout
+import Then
+import UIKit
+import WebKit
+
+class WebViewController: UIViewController {
```

Comment:
```suggestion
final class WebViewController: UIViewController {
```

상속하려는 ViewController가 아닌 이상 final 키워드를 꼭 붙여주세요.